server/src/group/group.service.ts

```diff
@@ -53,4 +54,46 @@ export class GroupService {
     const groupMember = await user.groupMember!.load();
     return groupMember!.group.load();
   }
+
+  /** Get group from the id. 
+   * Throws a NotFoundError if the id does not correspond to a group. */
+  async getGroupFromID(id: string): Promise<Group> {
+    const group = await this.groupsRepository.findOne({ friendlyId: id })
+    if (group == null)
+      throw NotFoundError
```

Comment:
I'm not sure if throwing an error is the best here, or if there is a better way to handle possible invalid ids. Could I just assume that all id's are correct as a precondition?